docs/work in progress/Auto triage issue - analysis.md

```diff
@@ -40,6 +40,28 @@
 3. It uses a simple Gated Recurrent Unit (GRU) based model architecture. 
 
     [https://github.com/machine-learning-apps/Issue-Label-Bot](https://github.com/machine-learning-apps/Issue-Label-Bot)
+    
+    
+    
+### VS Code Issue Triage Bot
+
+
+
+1. All recent issues are passed through two models that map the issue to a feature area and determine the assignee with an accuracy of roughly 75%.
+2. If the bot cannot verify an issue on its own, it labels it as author-verification-requested. Once verified, the bot labels the issue as verified.
+3. The bot supports commands that can close issues, add labels, remove labels, and add comments. 
+4. This is the basic triaging flow:
+![image](https://user-images.githubusercontent.com/84702817/123509847-7ec14a80-d695-11eb-9ae8-a9725fbceec7.png)
+5. The bot can close the issues for 14 reasons as listed here : [https://github.com/microsoft/vscode/wiki/Issues-Triaging#closing-issues](https://github.com/microsoft/vscode/wiki/Issues-Triaging#closing-issues)
```

Comment:
Can you please update the links?
Follow [item_name](link) syntax for putting links in .md Files. 
Reference : https://www.codegrepper.com/code-examples/whatever/how+to+insert+a+link+in+github+readme